python/lsst/ts/watcher/rules/under_pressure.py

```diff
@@ -0,0 +1,163 @@
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+__all__ = ["UnderPressure"]
+
+import yaml
+
+from .base_ess_rule import BaseEssRule
+
+
+class UnderPressure(BaseEssRule):
+    """Check for low pressure.
+
+    This rule only reads ESS telemetry topics.
+
+    Parameters
+    ----------
+    config : `types.SimpleNamespace`
+        Rule configuration, as validated by the schema.
+
+    Notes
+    -----
+    The alarm name is f"UnderPressure.{name}".
+
+    Like most rules based on data from the ESS CSC: this uses
+    `FilteredTopicField` and its ilk, because a given topic may be output
+    for more than one sensor (e.g. there may be two pressure sensors
+    or two 4-channel pressure sensors connected to the same CSC)
+    where the data is differentiated by the value of the sensorName field.
+    """
+
+    def __init__(self, config):
+        super().__init__(
+            config=config,
+            rule_name=f"UnderPressure.{config.name}",
+            topic_attr_name="tel_pressure",
+            field_name="pressure",
+            is_indexed=True,
+            big_is_bad=False,
+            units="Pa",
+            value_format="0.2f",
+            sensors=config.pressure_sensors,
+        )
+
+    @classmethod
+    def get_schema(cls):
+        schema_yaml = """
+$schema: http://json-schema.org/draft-07/schema#
+description: Configuration for UnderPressure rule.
+type: object
+properties:
+  name:
+    description: System being monitored, e.g. "MTM1M3 Compressor".
+    type: string
+  pressure_sensors:
+    description: >-
+        ESS pressure sensors to monitor.
+        Pressures are reported in the pressure telemetry topic.
+    type: array
+    minItems: 1
+    items:
+      type: object
+      properties:
+        sal_index:
+          description: SAL index of ESS CSC.
+          type: integer
+        sensor_info:
+          description: List of dict of sensor_name, indices.
+          type: array
+          minItems: 1
+          items:
+            type: object
+            properties:
+              sensor_name:
+                description: value of sensorName field.
+                type: string
+              indices:
+                description: >-
+                  Indices of the data to read (optional).
+                  If omitted then read all non-nan values.
+                type: array
+                items:
+                  type: integer
+            required:
+              - sensor_name
+            additionalProperties: false
+      required:
+        - sal_index
+        - sensor_info
+      additionalProperties: false
+  warning_level:
+    description: >-
+        The pressure (C) below which a warning alarm is issued.
```

Comment:
Here and in all descriptions below `C` is mentioned as unit for pressure. Shouldn't that be `Pa` (or whatever real unit is used for pressure)?